explorer_frontend/src/features/logs/hooks/useRepeatedGetTxByHash.ts

```diff
@@ -34,6 +34,7 @@ export const useRepeatedGetTxByHash = (
       } catch (err) {
         setError(true);
         setLoading(false);
+        clearInterval(intervalId);
```

Comment:
Why are you closing interval on error? We don't want retry?

Reply:
Yes, retry here makes a little sense, because if there is an error the reason is likely not to go by itself (becasue it is an error on backends side or error with connection). What i mean - we better display "Error while fetching tx details" in logs than spam with requests.
I don't see a case where retry can definitely help. If something is down on backend - it is not going to work in next 5-10 seconds with high probability. That's why i didn't add even 2-3 retries.

Reply:
Ok, i thought about it and decided that you are correct. I'll add 3 retries. Actually, the issue was that there were unlimited retires, but 3-5 of them is a decent idea.